install.sh

```
@@ -0,0 +1,415 @@
+    exit 1
+  fi
+  if ! wait_for_curia; then
+    error "Curia did not become healthy — refusing to print a success banner."
+    hint "Check logs: docker compose --project-directory \"$INSTALL_DIR\" logs curia"
+    exit 1
+  fi
+
+  # If the existing install used the domain topology, load DOMAIN into the shell
+  # so print_summary can show the correct HTTPS URL instead of localhost.
+  if grep -qE '^DOMAIN=.+' "$env" 2>/dev/null; then
+    DOMAIN="$(grep -E '^DOMAIN=' "$env" | cut -d= -f2-)"
```

Shellcheck (0.11.0)

[warning] DOMAIN appears unused. Verify use (or export if used externally).

(SC2034)